pkg/common/telemetry/specific_metrics.go

```diff
@@ -0,0 +1,428 @@
+package telemetry
+
+import (
+	"strconv"
+	"time"
+)
+
+// Call Counters (timing and success metrics)
+// Allows adding labels in-code
+
+// AddAttestorType add Attestor type label to the given counter
+// from the given attestor type
+func AddAttestorType(cc *CallCounter, aType string) {
+	cc.AddLabel(Attestor, aType)
+}
+
+// AddAudience add the Audience label(s) to the given counter
+// from the given audience(s)
+func AddAudience(cc *CallCounter, auds ...string) {
+	for _, aud := range auds {
+		cc.AddLabel(Audience, aud)
+	}
+}
+
+// AddCallerID add the CallerID label to the given counter
+// from the given ID
+func AddCallerID(cc *CallCounter, id string) {
+	cc.AddLabel(CallerID, id)
+}
+
+// AddRegistered add the Registered label to the given
+// counter, from given boolean
+func AddRegistered(cc *CallCounter, reg bool) {
+	cc.AddLabel(Registered, strconv.FormatBool(reg))
+}
+
+// AddSPIFFEID add SPIFFE ID label to the given counter
+// from the given ID
+func AddSPIFFEID(cc *CallCounter, id string) {
+	cc.AddLabel(SPIFFEID, id)
+}
+
+// ManagerFetchUpdatesSyncCall return a call counter for
+// the agent's manager fetching updates on sync
+func ManagerFetchUpdatesSyncCall(m Metrics) *CallCounter {
+	return StartCall(m, Manager, Sync, FetchUpdates)
+}
+
+// ManagerPruneBundleCall return a call counter
+// for manager bundle pruning
+func ManagerPruneBundleCall(m Metrics) *CallCounter {
```

Comment:
`Manager` is pretty generic. Will there only ever be one manager in SPIRE?
How about `BundleManagerPruneCall`?

Reply:
it is the CA manager, so will call it that